components/SetAction.coffee

```diff
@@ -0,0 +1,25 @@
+noflo = require 'noflo'
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.inPorts.add 'action',
+    datatype: 'string'
+    required: true
+    control: true
```

Comment:
Not really needed, was going to do it with process API first, but decided to stick to WirePattern until 0.8 stable is out